guides/concepts/list-of-operations.md

```diff
@@ -77,19 +77,65 @@ Possible errors:
+A path payment sends an amount of a specific asset to a destination account through a path of offers. Since the asset sent (e.g., 450 XLM) can be different from the asset received (e.g, 6 BTC), path payments allow for the simultaneous transfer and conversion of currencies.  
+
+A Path Payment Strict Send allows a user to specify the *amount of the asset to send*. The amount received will vary based on offers in the order books.  If you would like to instead specify the amount received, use the [Path Payment Strict Receive](#path-payment-strict-receive) operation.  
+
+A few things to note:
+* path payments don't allow intermediate offers to be from the source account as this would yield a worse exchange rate. You'll need to either split the path payment into two smaller path payments, or ensure that the source account's offers are not at the top of the order book.
+* balances are settled at the very end of the operation
+   * this is especially important when `(Destination, Destination Asset) == (Source, Send Asset)` as this provides a functionality equivalent to getting a no interest loan for the duration of the operation.
+
+Threshold: Medium
+
+Result: `PathPaymentStrictSendResult`
+
+Parameters:
+
+|Parameters| Type| Description|
+| --- | --- | --- |
+|Send asset| asset| The asset deducted from the sender's account.|
+|Send amount| integer| The amount of `send asset` to deduct (excluding fees).|
+|Destination| account ID| Account ID of the recipient.|
+|Destination asset| asset| The asset the destination account receives.|
+|Destination min| integer| The minimum amount of `destination asset` the destination account can receive.|
+|Path| list of assets| The assets (other than `send asset` and `destination asset`) involved in the offers the path takes. For example, if you can only find a path from USD to EUR through XLM and BTC, the path would be USD -> XLM -> BTC -> EUR and the `path` field would contain XLM and BTC.|
+
+Possible errors:
+
+| Error | Code | Description |
+| ----- | ---- | ------|
+|PATH_PAYMENT_STRICT_SEND_MALFORMED| -1| The input to this path payment is invalid.|
+|PATH_PAYMENT_STRICT_SEND_UNDERFUNDED| -2| The source account (sender) does not have enough funds to send and still satisfy its selling liabilities. Note that if sending XLM then the sender must additionally maintain its minimum XLM reserve.|
+|PATH_PAYMENT_STRICT_SEND_SRC_NO_TRUST| -3| The source account does not trust the issuer of the asset it is trying to send.|
+|PATH_PAYMENT_STRICT_SEND_SRC_NOT_AUTHORIZED| -4| The source account is not authorized to send this payment. |
+|PATH_PAYMENT_STRICT_SEND_NO_DESTINATION| -5| The receiving account does not exist. |
+|PATH_PAYMENT_STRICT_SEND_NO_TRUST| -6| The receiver does not trust the issuer of the asset being sent. For more, see the [assets doc](./assets.md).|
```

Comment:
Consider changing instances of "receiving account/receiver" to "destination account" in these lines for consistency with the operation fields.